plugins/providers/alicloudiam/config.go

```diff
@@ -0,0 +1,228 @@
+package alicloudiam
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/bearaujus/bptr"
+	"github.com/go-playground/validator/v10"
+	"github.com/goto/guardian/domain"
+	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	AccountTypeRamUser = "ramUser"
+	AccountTypeRamRole = "ramRole"
+)
+
+type Credentials struct {
+	AccessKeyID     string `mapstructure:"access_key_id" json:"access_key_id" validate:"required,base64"`
+	AccessKeySecret string `mapstructure:"access_key_secret" json:"access_key_secret" validate:"required,base64"`
+	RoleToAssume    string `mapstructure:"role_to_assume" json:"role_to_assume,omitempty"`
```

Comment:
let's change this to `RamRole` so it'll be consistent with the maxcompute and oss providers
```suggestion
	RoleName    string `mapstructure:"ram_role" json:"ram_role,omitempty"`
```

Reply:
Resolved: https://github.com/goto/guardian/pull/190/commits/6bd8fdffe0d3723f936bc84ddaf326d111b06d9b